app/Platform/Enums/AchievementSetType.php

```diff
@@ -47,4 +47,17 @@ enum AchievementSetType: string
     case WillBeBonus = "will_be_bonus";
     case WillBeSpecialty = "will_be_specialty";
     case WillBeExclusive = "will_be_exclusive";
+
+    public function label(): string
+    {
+        return match ($this) {
+            self::Core => 'Core',
+            self::Bonus => 'Bonus',
+            self::Specialty => 'Specialty',
+            self::Exclusive => 'Exclusive',
+            self::WillBeBonus => 'Bonus*',
+            self::WillBeSpecialty => 'Specialty*',
+            self::WillBeExclusive => 'Exclusive*',
```

Comment:
It makes sense now that I see this, but I was confused what the asterisk meant. Would it be worth adding a tooltip, or since this is a "temporary" thing, just disseminate the knowledge as we go?

Reply:
No harm in adding a tooltip for now. Added in latest:

![Screenshot 2024-09-26 at 6 09 37 PM](https://github.com/user-attachments/assets/ea560d07-ab32-457e-848a-13e7b3d95100)